README.md

```diff
@@ -26,11 +27,63 @@ public class Startup
 	public void Configuration(IAppBuilder app)
 	{
 		app.UseApplicationInsights();
+		
 		// rest of the config here...
 	}
 }
 ```
 
+Each operation will be provided with unique Guid Context.Operation.Id property by default. However, there
```

Comment:
I suggest you create a new section called "Passing OperationId via header" and place it after "Optional steps" and before "How this stuff works". At the beginning you should explain in what circumstances one would want to use this feature.